bin/plugin/commands/performance.js

```diff
@@ -147,30 +151,33 @@ async function getPerformanceResultsForBranch(
 	const results = [];
 	for ( let i = 0; i < 3; i++ ) {
 		await runShellScript(
-			'npm run test-performance',
+			`npm run test-performance -- packages/e2e-tests/specs/performance/${ testSuite }.test.js`,
 			performanceTestDirectory
 		);
 		const rawResults = await readJSONFile(
 			path.join(
 				performanceTestDirectory,
-				'packages/e2e-tests/specs/performance/results.json'
+				`packages/e2e-tests/specs/performance/${ testSuite }.test.results.json`
 			)
 		);
 		results.push( curateResults( rawResults ) );
 	}
 
-	return {
-		load: formatTime( median( results.map( ( r ) => r.load ) ) ),
-		domcontentloaded: formatTime(
-			median( results.map( ( r ) => r.domcontentloaded ) )
-		),
-		type: formatTime( median( results.map( ( r ) => r.type ) ) ),
-		minType: formatTime( median( results.map( ( r ) => r.minType ) ) ),
-		maxType: formatTime( median( results.map( ( r ) => r.maxType ) ) ),
-		focus: formatTime( median( results.map( ( r ) => r.focus ) ) ),
-		minFocus: formatTime( median( results.map( ( r ) => r.minFocus ) ) ),
-		maxFocus: formatTime( median( results.map( ( r ) => r.maxFocus ) ) ),
+	const medians = {
+		load: median( results.map( ( r ) => r.load ) ),
+		domcontentloaded: median( results.map( ( r ) => r.domcontentloaded ) ),
+		type: median( results.map( ( r ) => r.type ) ),
+		minType: median( results.map( ( r ) => r.minType ) ),
+		maxType: median( results.map( ( r ) => r.maxType ) ),
+		focus: median( results.map( ( r ) => r.focus ) ),
+		minFocus: median( results.map( ( r ) => r.minFocus ) ),
+		maxFocus: median( results.map( ( r ) => r.maxFocus ) ),
 	};
+
+	// Remove results for which we don't have data (and where the statistical functions thus returned NaN or Infinity etc).
+	const finiteMedians = pickBy( medians, isFinite );
```

Comment:
Does this has an impact on the `WPFormattedPerformanceResults` type? things should be made optional there right?

Reply:
I thought I'm doing that here? https://github.com/WordPress/gutenberg/pull/23842/files#diff-f7011184c69b1a3bacc4458a3cc16446R51-R58 :sweat_smile: :thinking:

Reply:
missed this somehow 😬 😄